dbfit-java/core/src/main/java/dbfit/api/DbCommand.java

```diff
@@ -0,0 +1,25 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * An object representing an executable database operation
+ * with no result returned.
```

Comment:
This condition is actually violated by `FunctionResultSetStatementExecution` as this is used for calling functions.